SpatialGDK/Source/SpatialGDK/Private/Utils/EntityFactory.cpp

```diff
@@ -262,13 +265,13 @@ TArray<FWorkerComponentData> EntityFactory::CreateEntityComponents(USpatialActor
 		{
 			check(NetDriver->VirtualWorkerTranslator != nullptr);
 
-			VirtualWorkerId IntentVirtualWorkerId = NetDriver->VirtualWorkerTranslator->GetLocalVirtualWorkerId();
```

Comment:
As this always used to return GetLocalVirtualWorkerId(), was this logic previously broken (or at least out of sync for a replication beat)?

Reply:
I think yea it must've been initially out of sync